app/controllers/teams/users_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Teams::UsersController < AuthenticatedController
+   before_action -> { authorize team, policy_class: TeamUsersPolicy }
+
+    def remove_confirm
+      @team = team
+      @user = team.users.find_by(id: params[:id])
+      if @user.present? && @user != current_user
```

Comment:
Similarly to the above, we could pull this logic out into a method, since we're using the same logic again below in 'destroy'